src/Middlewares/ErrorHandler.js

```diff
@@ -0,0 +1,5 @@
+const ErrorHandler = (Error, req, res, next) => {
```

Comment:
```suggestion
const ErrorHandler = (error, req, res, next) => {
```
`Error` kelimesi JS'te hata sınıfına verildiği için "e" harfinin küçük olması karışıklık olmaması adına daha doğru olacaktır